host-bmc/host_pdr_handler.cpp

```diff
@@ -1428,8 +1432,8 @@ void HostPDRHandler::setLocationCode(
                         /* Remove this once phyp support is in place */
                         if (node.entity_type == PLDM_ENTITY_SYSTEM_CHASSIS)
                         {
-                            CustomDBus::getCustomDBus().setSoftwareVersion(
-                                entity.first, "sample_mex_version");
+                            // CustomDBus::getCustomDBus().setSoftwareVersion(
+                            //     entity.first, "sample_mex_version");
```

Comment:
add the support for this as well george.

Reply:
> add the support for this as well george.

Done